test/ng/compileSpec.js

```diff
@@ -7427,6 +7459,51 @@ describe('$compile', function() {
       });
     });
 
+
+    it('should be possible to change the scope of a directive using $provide', function() {
+      module(function($provide) {
+        directive('foo', function() {
+          return {
+            scope: {},
+            template: '<div></div>'
+          };
+        });
+        $provide.decorator('fooDirective', function($delegate) {
+          var directive = $delegate[0];
+          directive.scope.something = '=';
+          directive.template = '<span>{{something}}</span>';
+          return $delegate;
+        });
+      });
+      inject(function($compile, $rootScope) {
+        element = $compile('<div><div foo something="abc"></div></div>')($rootScope);
+        $rootScope.abc = "bar";
```

Comment:
I meant _double quotes_ --> _single quotes_ (not _abc_ ---> _bar_).

But nevermind. My battle against double/single quote consistency is a lost cause anyway :stuck_out_tongue: 
They're everywhere...